xdsl/dialects/stablehlo.py

```diff
@@ -342,13 +352,17 @@ def print_parameters(self, printer: Printer) -> None:
     def parse_parameters(cls, parser: AttrParser) -> Sequence[Attribute]:
         with parser.in_angle_brackets():
             lhs_batching_dimensions = DotAttr._parse_parameter(
-                "lhs_batching_dimensions", parser
-            )
-            parser.parse_punctuation(",")
-            rhs_batching_dimensions = DotAttr._parse_parameter(
-                "rhs_batching_dimensions", parser
+                "lhs_batching_dimensions", parser, optional=True
             )
-            parser.parse_punctuation(",")
+            if lhs_batching_dimensions != ArrayAttr(()):
```

Comment:
Same deal here, seems a little clearer to me:
```suggestion
            if lhs_batching_dimensions.data:
```